### DISCUSSION TITLE
Can SE get sample color palette from picture/video

### DISCUSSION THREAD

**Discussion by henryhaseo** [SubtitleEdit/subtitleedit#5719] [Q&A] [answered]
Is there a way to get sample color palette from picture/video? Because SE use 7 code and Hex code only have 6 code.

**Comment by niksedk** [comment]
Could you explain a bit more? Screenshots welcome.

**Reply by henryhaseo** [reply]
If i use Hex Color #1902D4 the color should be Blue, but in SE the color is Red.

![Untitled2](https://user-images.githubusercontent.com/64903912/149879186-a6a8d18e-54ad-41c5-a5bc-274495df469c.png)

![Untitled](https://user-images.githubusercontent.com/64903912/149879556-63ced7c3-b753-4659-9d50-265c306dbbc7.png)

How can i take the sample color from the video/picture to use in SE?

thanks

**Comment by niksedk** [comment, marked as answer]
Your blue color will look like this in ASSA ("syntax coloring" enabled via options - settings - appearance):
![image](https://user-images.githubusercontent.com/1261331/149886527-e6fc2055-e577-4037-8015-fc494baf3889.png)

Note that ASSA uses "BGR color" and not "RGB" color (R=red, G=green, B=blue). and alpha is also opposite of what is normal.

No, SE does not have a color picker.

**Reply by henryhaseo** [reply]
Thank you very much for your answer. I just need to change the code from RGB(1902D4) to BGR(D40219) to get the same color i need.

**Comment by niksedk** [comment]
Also, the "Assa Draw" plugin now has a color picker
